aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -139,6 +139,25 @@ func TestAccAWSElasticacheCluster_Engine_Redis_Ec2Classic(t *testing.T) {
 	})
 }
 
+func TestAccAWSElasticacheCluster_Port_Redis_Default(t *testing.T) {
+	var ec elasticache.CacheCluster
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSElasticacheClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSElasticacheClusterConfig_RedisDefaultPort,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSElasticacheClusterExists("aws_elasticache_cluster.foo", &ec),
```

Comment:
Our convention is to use `test`, `test1`, etc. instead of `foo` and `bar` now. Would you update these?

Reply:
I went ahead and cleanup the whole file! I reran most of the tests in that file locally, so I don't think I missed any spots. But it might be worth another scan to make sure. 😄

Reply:
@aeschright